### PR TITLE
Add Rift build

### PR DESCRIPTION
**Pull request by mikeage**

### CODE REVIEW

**Comment by mikeage** [comment]
(draft because not even pre-alpha tested yet!)

File: Assets/Editor/BuildWindow.cs [hunk 1]
```diff
@@ -472,7 +472,7 @@ private class AndroidOperation {
     private void OnBuildSettingsChanged()
     {
       // Only set this if supporting SDK that needs Android (and is installed!).
-#if OCULUS_SUPPORTED
+#if UNITY_ANDROID
```
Comment (mikeage) [inline Assets/Editor/BuildWindow.cs:475]: Followup to #92

**Comment by mikeskydev** [comment]
Works great using Quest 1 and Link! Will test later tonight with Rift CV1 when available.

**Comment by mikeage** [comment]
Awesome! I'm going to rebase now to take the new Oculus package just to be sure; please get the newer build when ready.

This was really easy ;-)

**Comment by mikeage** [comment]
@mikeskydev should we do a Rift Experimental build and also push to the release page, or is this a bit more niche and really only meant for the Oculus store?

**Comment by mikeskydev** [comment]
I think we're okay to skip out on the experimental right now, if Rift users are interested they'll always have the SteamVR powered one to try out! We should also consider that there's technically no desktop Oculus HMD in production anymore, aside from linking via Quest which may still be a bit niche for the average user. I think if we see any specific demand for it, we can just add it later.

**Comment by mikeage** [comment]
Ok, great. So this is ready to go if/when your testing is complete!

**Review by mikeskydev — APPROVED** [review]
LGTM! There's an issue where the Rift CV1 controller models are invisible, but this also happens in the SteamVR version so isn't related to the SDK most likely. Happy to merge this now, and sort the issue in a separate PR. 
image

**Pull request merged**